SpatialGDK/Documentation/content/get-started/gdk-and-starter-project.md

```diff
@@ -0,0 +1,63 @@
+# Get started: 3 - Set up the Starter Project Respository and the Unreal GDK Repository
+
+### Clone the Repositories
+
+In this step, you are going to clone the Starter Project repository and then clone the GDK repository into your copy of the Starter Project. Note that the GDK needs to go into the Starter Project's `Plugins` directory.
+
+1. Clone the [Starter Project](https://github.com/spatialos/UnrealGDKStarterProject/) repository by running either:
+    * (HTTPS) `git clone https://github.com/spatialos/UnrealGDKStarterProject.git`
+    * (SSH) `git clone git@github.com:spatialos/UnrealGDKStarterProject.git`
+2. Navigate into `<StarterProjectRepoRoot>\Game\Plugins`. You may need to create the Plugins folder if you don't have any plugins installed yet.
+3.  In a Git Bash terminal window, clone the [Unreal GDK](https://github.com/spatialos/UnrealGDK) repository by running either:
+    * (HTTPS) `git clone https://github.com/spatialos/UnrealGDK.git`
```

Comment:
Are we pinning to a specific tag of this? Right now everyone will just get `master`...

Reply:
I spoke with the rest of the team, they are for not pinning, as they fear people will not be prompted to unpin and won't receive bug-fixes and new features in a timely fashion.

This certainly isn't the last word on the matter, as we need to define and recommend the best workflow for users to keep up to date with master, but I'm not going to make any changes to his PR.

Reply:
I think getting feedback from some of the Unity team is valuable here @callumlawson @jamiebrynes7 @jessicafalk 
The fear is quite hypothetical, and easily corrected if we find that people aren't upgrading themselves.
I'd rather we follow a generally good practice from the get-go, and if/when it's shown that it's not having the effect we want, THEN change it.

Reply:
Its probably fine not to pin if we swap to a `develop` branch and have `master` be pinned to latest release

Reply:
The main reason why we decided to pin is because of the starter project. We can't always promise that both master branches (of the gdk-for-unity and the gdk-for-unity-fps-starter-project) are always compatible with each other. Especially when we e.g. upgrade the worker snapshot or some API that the starter project uses.

Reply:
Ultimately the workflows will differ quite a bit once 3rd party support for the UPM is in and you don't have to sideload anymore